src/db.ts

```diff
@@ -1,9 +1,25 @@
-import { query } from '@rise-tools/server'
+import { PrismaClient, Video } from '@prisma/client'
+import { lookup, query } from '@rise-tools/server'
 
-export type Content = {
-  title: string
-  thumbnail: string
-  id: string
-}[]
+export const prisma = new PrismaClient()
 
-export const videos = query<Content>(() => require('./data/2023.json'))
+/**
+ * Model to get all videos from a given edition
+ */
+export const edition = lookup((edition: string) =>
```

Comment:
we should consider keeping all the models in one file. Then the `prisma.video.update` and invalidations could happen directly inside the onSubmit handler, to really help people see how simple this code is

Reply:
you mean to keep them together with admin panel, for readability? 

that sounds good! this won't be true for conference app, where we want to hide that fact on purpose, but we will just import from admin file!

Reply:
Good plan!!